Makefile.am

```diff
@@ -1370,6 +1370,14 @@ network_la_CPPFLAGS += $(GCRYPT_CPPFLAGS)
 network_la_LDFLAGS += $(GCRYPT_LDFLAGS)
 network_la_LIBADD += $(GCRYPT_LIBS)
 endif
+
+test_plugin_network_SOURCES = src/network_test.c \
+			      src/daemon/configfile.c \
+			      src/daemon/types_list.c
+test_plugin_network_CPPFLAGS = $(AM_CPPFLAGS)
```

Comment:
This is not needed since you don't override the flags.

Reply:
Fixed, thanks!